src/graphics/Screen.cpp

```diff
@@ -1600,6 +1639,14 @@ void Screen::handleSetOn(bool on, FrameCallback einkScreensaver)
             pinMode(VTFT_LEDA, OUTPUT);
             digitalWrite(VTFT_LEDA, TFT_BACKLIGHT_ON);
 #endif
+#endif
+#if defined(M5STACK_COREBASIC)
+            pinMode(TFT_BL, OUTPUT);
+            digitalWrite(TFT_BL, HIGH);
+#endif
```

Comment:
TFT_BL should be handled out of the box if defined. If it doesn't, please bugfix instead ;-)

Reply:
TFT_BL has been defined in variant.h, but I found that if this one is not set, the screen backlight is not turned on when the button wakes up the device, and I think this should apply to other devices